### DISCUSSION TITLE
uBlock is not blocking shopping items from google search result

### DISCUSSION THREAD

**Discussion by ham3dt** [uBlockOrigin/uAssets#21478] [Ideas]
### Prerequisites

- [X] This is **NOT** a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL(s) where the issue occurs.

`https://www.google.com/search?q=400w+buck+converter`

### Description

The sponsored result are blocked but the I want to remove the shopping lists and the side panel.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![Capture](https://github.com/uBlockOrigin/uAssets/assets/25421550/bab9c41a-d3f4-4f8a-9170-0ee61f02b208)

</details>

### Configuration

<details>
<summary>Details</summary>

uBlock Origin: 1.54.0
Firefox: 119
filterset (summary):
 network: 290788
 cosmetic: 244829
 scriptlet: 49451
 html: 1359
listset (total-discarded, last-updated):
 added:
  adguard-generic: 81349-6969, 1d.12h.11m
  adguard-mobile: 8918-142, 1d.12h.11m
  adguard-spyware-url: 1360-92, 1d.12h.11m
  adguard-spyware: 80469-17070, 1d.12h.12m
  block-lan: 55-0, 21d.15h.17m
  curben-phishing: 79022-29, 23h.52m
  adguard-social: 22098-875, 1d.12h.12m
  adguard-cookies: 28247-62, 1d.12h.12m
  ublock-cookies-adguard: 772-0, 12h.25m
  adguard-popup-overlays: 26618-1586, 1d.12h.13m
  adguard-mobile-app-banners: 4847-1070, 1d.12h.13m
  adguard-other-annoyances: 14057-332, 1d.12h.13m
  [12 lists not shown]: [too many]
 default:
  user-filters: 13-0, never
  ublock-filters: 37318-595, 59m
  ublock-badware: 7637-11, 59m
  ublock-privacy: 945-4, 59m
  ublock-unbreak: 2218-2, 59m
  ublock-quick-fixes: 116-0, 59m
  easylist: 75866-1019, 59m
  easyprivacy: 33117-987, 59m
  urlhaus-1: 11620-0, 40m
  plowe-0: 3777-1, 8d.15h.8m
filterset (user): [array of 10 redacted]
trustedset:
 added: [array of 1 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1200 ms (selfie)
 maxAssetCacheWait: 653 ms
popupPanel:
 blocked: 54
 network:
  google.com: 39
  doubleclick.net: 3
  youtube.com: 12
 extended:
  ###tadsb[aria-label]
  ##.cu-container
  ##.uEierd
  ##a[href^="https://www.googleadservices.com/pagead/aclk?"]
  ###bottomads > div[id^="tads"]
  ##div[aria-label="Why this ad?"]:upward([data-pla="1"])
  ##+js(set-constant, rwt, noopFunc)
  ##+js(trusted-set-cookie, SOCS, CAESHAgBEhJnd3NfMjAyMzA2MTItMF9S…
  ##+js(remove-attr, ping, div[id="search"] a[data-ved][ping])
  ##.ytp-cards-teaser
  ##.ytp-paid-content-overlay
  ##.ytp-chrome-top-buttons > .ytp-cards-teaser
  ###player [class*="suggested-action-badge-icon"]:upward(.ytp-sug…
  ##+js(trusted-replace-xhr-response, /"adPlacements.*?([A-Z]"\}|"…
  ##+js(trusted-replace-xhr-response, /"adPlacements.*?("adSlots"|…
  ##+js(trusted-replace-fetch-response, /"adPlacements.*?([A-Z]"\}…
  ##+js(trusted-set-cookie, SOCS, CAISNQgDEitib3FfaWRlbnRpdHlmcm9u…
  ##+js(set-constant, ytInitialPlayerResponse.auxiliaryUi.messageR…
  ##+js(json-prune, auxiliaryUi.messageRenderers.upsellDialogRende…
  ##+js(set-constant, ytInitialPlayerResponse.playerAds, undefined…
  ##+js(set-constant, ytInitialPlayerResponse.adPlacements, undefi…
  ##+js(set-constant, ytInitialPlayerResponse.adSlots, undefined)
  ##+js(set-constant, playerResponse.adPlacements, undefined)
  ##+js(set-constant, google_ad_status, 1)
</details>

**Comment by Daxy7** [comment]


**Reply by ham3dt** [reply]
You are literally showing the side panel in the picture and saying you're not seeing it?

**Comment by D4niloMR** [comment]
Try:
```
www.google.com###rcnt [jscontroller] > div.sh-dr__restricts[data-drrc]
www.google.com##[role="navigation"] a[href*="&tbm=shop&"]
```

**Reply by ham3dt** [reply]
It has no effect.

**Reply by D4niloMR** [reply]
What are you trying to remove? Screenshot it please.

**Reply by ham3dt** [reply]
Read the first post description.

**Comment by ham3dt** [comment]
I fixed it by adding these filters:
```
! 2023-12-21 https://www.google.com
www.google.com##.T98FId
www.google.com##.wH6SXe
www.google.com##.VlIwlf
www.google.com##.O4T6Pe
www.google.com###appbar

! 2023-12-21 https://www.google.com
www.google.com##.RDi0y:style(margin-left: 10% !important)
www.google.com##.C9KyDd.FEoSJf
```
But I think there should be a better way...

**Reply by Daxy7** [reply]
> I fixed it by adding these filters:

none of these filters work for me

the side panel is `##.sh-dr__restricts` and the top panel is `##.w8oiwc` so idk. ![clueless](https://github.com/uBlockOrigin/uAssets/assets/91396482/e4fce598-1ecc-45fd-8809-c00b44ca9523)

**Comment by ham3dt** [comment]
This is what I get, a clean search result:
![Capture](https://github.com/uBlockOrigin/uAssets/assets/25421550/90cf69ac-14f4-42c3-a1ba-f7b21fd71417)

But notice how the "All filters" menu pushed to the right? it made an horizontal scroll to appear!

**Reply by Daxy7** [reply]
in your screenshot(s) at the submitted issue, you're in the [shopping section,](https://www.google.com/search?q=shopping&tbm=shop) but the url is `https://www.google.com/search?q=400w+buck+converter`, what are you trying to block in the standard search results?
```
! add your comment here
www.google.com##.T98FId

! add your comment here
www.google.com##.wH6SXe

! add your comment here
www.google.com##.VlIwlf

! add your comment here
www.google.com##.O4T6Pe

! add your comment here
www.google.com###appbar

! add your comment here
www.google.com##.RDi0y:style(margin-left: 10% !important)

! add your comment here
www.google.com##.C9KyDd.FEoSJf
```

**Reply by ham3dt** [reply]
That's the problem, I'm NOT in the shopping section and yet I'm getting shopping results.

Here's another SS:
![Untitled](https://github.com/uBlockOrigin/uAssets/assets/25421550/8ee115f0-11eb-4946-a1f1-698e59c2f5c9)

The search results are right after MANY shopping suggestion at the bottom of the page.

**Reply by Daxy7** [reply]
i get half of your results signed in but not the two panels and minimized to fit in with website reuslts

![image](https://github.com/uBlockOrigin/uAssets/assets/91396482/77c0fa3f-58c8-4491-9963-f1c92a5ae0fe)

**Reply by ham3dt** [reply]
I don't know why your results differ from me, but maybe something is changed on the google side cause I'm having this problem very recently.

**Reply by krystian3w** [reply]
Need US 🇺🇸  IP?

**Reply by ham3dt** [reply]
I tried it with 3 different IPs, it's not location wise.

**Reply by krystian3w** [reply]
Works after you logout and clear cookies?

**Reply by ham3dt** [reply]
No difference.

**Reply by krystian3w** [reply]
Can you try in Firefox Portable?

https://portableapps.com/apps/internet/firefox_portable

**Reply by ham3dt** [reply]
I don't think the problem is browser based, either way I only download things from official sources.

**Reply by krystian3w** [reply]
So try fresh Firefox profile:

https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles

**Reply by ham3dt** [reply]
How on earth google results is related to my browser?
